src/leo_redundant_manager_worker.erl

```diff
@@ -790,20 +828,20 @@ get_node_by_vnode_id(TableInfo, VNodeId) ->
 
 %% @doc Retrieve a member from an argument.
 %% @private
--spec(get_redundancies([#member{}], atom(), [string()]) ->
-             {atom, [string()]} | not_found).
+-spec(get_redundancies([{Node, RackId}], atom(), [string()]) ->
+             {atom, [string()]} | not_found when Node::node(),
+                                                 RackId::string()).
 get_redundancies([],_,_) ->
     not_found;
-get_redundancies([#member{node = Node_0,
-                          grp_level_2 = L2}|_], Node_1, SetL2) when Node_0 == Node_1 ->
-    case lists:member(L2, SetL2) of
+get_redundancies([{Node_0, NumOfRAs}|_], Node_1, RackIdL) when Node_0 == Node_1 ->
```

Comment:
NumOfRAs is not suitable variable name as it holds the RackId of Node_0 so I suggest to rename it to something like RackId or Node_0_RackId.